README.md

```diff
@@ -1,2 +1,43 @@
 # sentry-fullstory
 The Sentry-FullStory integration creates a link from the Sentry Error to the FullStory replay. It also creates a link from the FullStory event to the Sentry error.
```

Comment:
Nit: `Sentry Error` - I'm guessing `Error` should be lowercase. If it's intended to be uppercase, then `Sentry error` should updated with consistent casing.